src/components/Header/Header.tsx

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import {
+  Box,
+  Flex,
+  HStack,
+  Spacer,
+  useMultiStyleConfig,
+  VStack,
+} from "@chakra-ui/react";
+
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import Icon from "../Icons/Icon";
+import { IconAlign, IconNames, IconSizes } from "../Icons/IconTypes";
+import Link from "../Link/Link";
+import List from "../List/List";
+import { ListTypes } from "../List/ListTypes";
+import Logo from "../Logo/Logo";
+import { LogoNames, LogoSizes } from "../Logo/LogoTypes";
+import HorizontalRule from "../HorizontalRule/HorizontalRule";
+import Notification from "../Notification/Notification";
+
+const Header = () => {
+  const styles = useMultiStyleConfig("Header", {});
+  const [width, setWidth] = React.useState(window.innerWidth);
+  const breakpointLarge = 960;
+  const breakpointMedium = 600;
+
+  React.useEffect(() => {
+    const handleWindowResize = () => setWidth(window.innerWidth);
+    window.addEventListener("resize", handleWindowResize);
+
+    return () => window.removeEventListener("resize", handleWindowResize);
+  }, []);
```

Comment:
Check out the `useWindowSize` hook in this repo. That can probably be used here to get the `width` value.